go/exorelay/exorelay_test.go

```diff
@@ -16,26 +16,26 @@ import (
 	"github.com/Originate/exocom/go/structs"
 )
 
+func newExocom() *exocomMock.ExoComMock {
+	exocom := exocomMock.New()
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}()
+	defer exocom.Close()
```

Comment:
I think `defer` closes the exocomMock instance once this method returns. Is that what we want here?